Assets/Scripts/Player/UnitGridCombat.cs

```diff
@@ -429,5 +430,9 @@ private enum Direction {
 			animator_.SetBool("isShootingLeft", false);
 			animator_.SetBool("isShootingRight", false);
 		}
+
+		public override string GetName() {

```

Comment:
Why do you use `string` here while in both the name variable and in the original function declaration you use `String`?